src/components/pages/admin/PageProductImport/components/CSVFileImport.tsx

```diff
@@ -31,25 +31,28 @@ export default function CSVFileImport({url, title}: CSVFileImportProps) {
   };
 
   const uploadFile = async (e: any) => {
-      // Get the presigned URL
-      const response = await axios({
-        method: 'GET',
-        url,
-        params: {
-          filename: encodeURIComponent(file.name)
-        }
-      });
-      const signedUrl = response.data.url;
-      console.log('File to upload: ', file.name)
-      console.log('Uploading to: ', signedUrl)
-      const result = await fetch(signedUrl, {
-        method: 'PUT',
-        body: file
-      })
-      console.log('Result: ', result)
-      setFile('');
-    }
-  ;
+    const token = localStorage.getItem('auth_token')
+    // Get the presigned URL
+    const response = await axios({
+      method: 'GET',
+      url,
+      params: {
+        filename: encodeURIComponent(file.name),
+      },
+      headers: {
+        Authorization: `basic ${token}`,
```

Comment:
No need to write "basic" 
See the generator for the header
https://www.blitter.se/utils/basic-authentication-header-generator/

Reply:
I don't think that it matters. The current authentication approach is far from ideal)